op-guide/ansible-deployment-rolling-update.md

```diff
@@ -20,20 +20,22 @@ When you perform a rolling update for a TiDB cluster, the service is shut down s
 
 1. Edit the value of the `tidb_version` parameter in the `/home/tidb/tidb-ansible/inventory.ini` file, and specify the version number you need to upgrade to.
 
-    For example, to upgrade from `v2.0.2` to `v2.0.3`:
+    For example, to upgrade from `v2.0.6` to `v2.0.7`:
 
     ```
-    tidb_version = v2.0.3
+    tidb_version = v2.0.7
     ```
 
+    > **Note:** If you use the master branch of `tidb-ansible`, you can keep `tidb_version = latest`. The installation package of the latest TiDB version is updated each day.
```

Comment:
If you use the master branch of `tidb-ansible`
->
If you use `tidb-ansible` of the master branch